src/overlays/actors/ovl_En_Ex_Ruppy/z_en_ex_ruppy.c

```diff
@@ -5,6 +5,7 @@
 #define FLAGS 0x00000010
 
 #define THIS ((EnExRuppy*)thisx)
+#define DIVING_GAME ((EnDivingGame*)this->actor.parent)
```

Comment:
dont think this macro is even really worth personally, its good information to know that its from parent without hiding it behind a macro

Reply:
also this from the original review of ex_ruppy
![hjdfg](https://user-images.githubusercontent.com/22045047/101829240-b8b76e80-3b00-11eb-9010-5e3585c4784c.png)

Reply:
Fair enough.